app/package.json

```diff
@@ -63,7 +63,7 @@
     "query-string": "7.0.1",
     "react": "17.0.2",
     "react-i18next": "11.13.0",
-    "react-native": "0.66.1",
+    "react-native": "^0.66.5",
```

Comment:
I think the dependency needs to match exactly the one from bifold. If using tilde or caret expression, they could still  diverge. Depending when `npm install` was ran it could end up with different versions.
I had provided a [script](https://github.com/bcgov/bc-wallet-mobile/blob/main/DEVELOPER.md#updating-packagejson) to automate the syncing of the package files ad the dependencies should always be exact the same.